package.json

```diff
@@ -69,9 +69,9 @@
     "pretest": "npm run lint",
     "prepublishOnly": "npm run build",
     "lint": "npm run lint:eslint && npm run lint:prettier",
-    "lint:eslint": "eslint '**/*.ts'",
-    "lint:prettier": "prettier --check '**/*{md,js,json,ts}'",
-    "format": "prettier --write '**/*{md,js,json,ts}'",
+    "lint:eslint": "eslint **/*.ts",
```

Comment:
I don't think this file should've changed. Was it not working for you?

Reply:
It was not working for me.

Reply:
Hmm, okay. That's something we should fix, but not in this pull request. I created #227 to note this, and I'll do it today.

Reply:
Scripts should now work.